crates/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -358,24 +343,38 @@ pub fn lock_manifest(
     pkgdb: &Path,
     manifest_path: &Path,
     existing_lockfile_path: Option<&Path>,
+    global_manifest_path: &Path,
 ) -> Result<serde_json::Value, EnvironmentError2> {
     let canonical_manifest_path = manifest_path
         .canonicalize()
         .map_err(EnvironmentError2::OpenManifest)?;
+
     let mut pkgdb_cmd = Command::new(pkgdb);
     pkgdb_cmd
-        .args(["manifest", "lock", "--ga-registry"])
-        .arg(canonical_manifest_path);
+        .args(["manifest", "lock"])
+        .arg("--ga-registry")
+        .arg("--global-manifest")
+        .arg(global_manifest_path);
     if let Some(lf_path) = existing_lockfile_path {
         let canonical_lockfile_path = lf_path
             .canonicalize()
             .map_err(EnvironmentError2::BadLockfilePath)?;
-        pkgdb_cmd.arg(canonical_lockfile_path);
+        pkgdb_cmd.arg("--lockfile").arg(canonical_lockfile_path);
     }
-    debug!(target: "posix", "locking manifest with command: {pkgdb_cmd:?}");
+    pkgdb_cmd.arg(canonical_manifest_path);
+
+    debug!("locking manifest with command: {pkgdb_cmd:?}");
     let output = pkgdb_cmd.output().map_err(EnvironmentError2::PkgDbCall)?;
     // If command fails, try to parse stdout as a PkgDbError
     if !output.status.success() {
+        eprintln!(
+            "pkgdb stdout content: {}",
+            String::from_utf8_lossy(&output.stdout)
+        );
+        eprintln!(
+            "pkgdb stderr content: {}",
+            String::from_utf8_lossy(&output.stderr)
+        );
```

Comment:
I guess should be general output on stdout and **error**\* messages on stderr?
\* errors since its non-successful not because its std*err*

Not sure what exactly will be on these actually though..

```suggestion
        debug!(
            "pkgdb stdout content: {}",
            String::from_utf8_lossy(&output.stdout)
        );
        error!(
            "pkgdb stderr content: {}",
            String::from_utf8_lossy(&output.stderr)
        );
```

Reply:
This is actually a mistake, this was left in while I was debugging something